docs/index.md

```diff
@@ -8,19 +8,19 @@
 </p>
 <p align="center">
 
-<a href="https://badge.fury.io/py/ludwig" target="_blank">
+<a href="https://badge.fury.io/py/ludwig" target="_blank" style="text-decoration: none;">
```

Comment:
Sorry I'm late to this PR! This is a great way to fix this issue given the scope of the problem currently.

If we find that we need to continue removing default styling on anchor tags, I would recommend making a class and including a custom stylesheet so we don't have to copy/paste this styling everywhere.

Reply:
Great idea!

Definitely happy to define a class in our existing custom CSS file if we find that we need to continue removing default styling on anchor tags.